content/pages/work-with-us.html

```diff
@@ -73,9 +73,7 @@ <h2>Transform the way you work</h2>
         {{ ask-qs | markdownify }}
       </div>
       <div class="tablet-lg:grid-col-5">
-        <img src="{{ site.baseurl }}/assets/img/work-with-us/work-with-us-illo-2.svg"
```

Comment:
Note to self to double-check this — I'm used to seeing things that replace {{ site.baseurl }} to have a `| url` filter at the end.

Reply:
Just checking, while I'm working down the list: since [this markup's been converted](https://github.com/18F/18f.gsa.gov/blob/6def599c34fdaee3d47e5199af3682fd9607ee7e/content/pages/work-with-us.html#L77), do we still need some action here, ?

Reply:
Hmmm, this looks like a merge conflict waiting to happen, but the big green button doesn't indicate any problems? So we're probably fine??